scripts/ci_before_init_embed.bash

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-export WEBOTS_RELEASE_VERSION=2023a
+export WEBOTS_RELEASE_VERSION=2022b-rev1
```

Comment:
The develop branch should be tested against 2023a nightlies, this shouldn't be changed